helm lint fails across multiple charts

```
+ helm lint emulator/ etamay/ modbus/ snmp/ synse-server/

==> Linting emulator/

Lint OK


==> Linting etamay/

[ERROR] Chart.yaml: apiVersion is required


==> Linting modbus/

[ERROR] Chart.yaml: apiVersion is required


==> Linting snmp/

[ERROR] Chart.yaml: apiVersion is required


==> Linting synse-server/

Lint OK


Error: 5 chart(s) linted, 3 chart(s) failed

script returned exit code 1
```

I think this may be a result of an updated version of helm which is more strict on the requirements for what goes in a `Chart.yaml`

--- a/etamay/Chart.yaml
+++ b/etamay/Chart.yaml
@@ -1,3 +1,4 @@
+apiVersion: v1
 name: etamay
 version: 1.0.0
 appVersion: 0.1.0

--- a/modbus/Chart.yaml
+++ b/modbus/Chart.yaml
@@ -1,5 +1,4 @@
----
-
+apiVersion: v1
 name: modbus
 version: 0.2.1
 appVersion: 1.1.1

--- a/snmp/Chart.yaml
+++ b/snmp/Chart.yaml
@@ -1,5 +1,4 @@
----
-
+apiVersion: v1
 name: snmp
 version: 0.1.1
 appVersion: 1.1.1

